ui/components/Header.js

```diff
@@ -112,7 +112,7 @@ const styles = (theme) => ({
   headerIcons : { fontSize : "1.5rem", height : "1.5rem", width : "1.5rem" },
   cbadge : {
     fontSize : "0.65rem",
-    backgroundColor : "white",
+    backgroundColor : "#ffffffd4",
```

Comment:
is this right color @AkashKumar7902 🤔

Reply:
@Yashsharma1911 The color is slightly lighter than that of the page title background as asked in the issue description.

Reply:
Ohh Thanks for explaining I thought It was wrong